### PR TITLE
edit NEWS for 1.11

### PR DESCRIPTION
**Pull request by JeffBezanson**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,94 +1,91 @@
...
 ---------------------------
 
 * The entry point for Julia has been standardized to `Main.main(ARGS)`. This must be explicitly opted into using the `@main` macro
-(see the docstring for further details). When opted-in, and julia is invoked to run a script or expression
-(i.e. using `julia script.jl` or `julia -e expr`), julia will subsequently run the `Main.main` function automatically.
-This is intended to unify script and compilation workflows, where code loading may happen
-in the compiler and execution of `Main.main` may happen in the resulting executable. For interactive use, there is no semantic
-difference between defining a `main` function and executing the code directly at the end of the script ([50974]).
+  (see the docstring for further details). When opted-in, and julia is invoked to run a script or expression
+  (i.e. using `julia script.jl` or `julia -e expr`), julia will subsequently run the `Main.main` function automatically.
```
Comment (aviatesk) [inline NEWS.md:45]:
```suggestion
  (see the docstring for further details). When opted-in, and `julia` is invoked to run a script or expression
  (i.e. using `julia script.jl` or `julia -e expr`), `julia` will subsequently run the `Main.main` function automatically.
```

File: NEWS.md [hunk 2]
```diff
@@ -1,94 +1,91 @@
...
   easier ([#43845]).
-* `@time` now reports a count of any lock conflicts where a `ReentrantLock` had to wait, plus a new macro
-  `@lock_conflicts` which returns that count ([#52883]).
 
 Language changes
 ----------------
-* During precompilation, the `atexit` hooks now run before saving the output file. This
+
+* During precompilation, `atexit` hooks now run before saving the output file. This
   allows users to safely tear down background state (such as closing Timers and sending
```
Comment (aviatesk) [inline NEWS.md:23]:
```suggestion
  allows users to safely tear down background state (such as closing `Timers` and sending
```

File: NEWS.md [hunk 3]
```diff
@@ -222,3 +202,62 @@ Tooling Improvements
...
+[#50074]: https://github.com/JuliaLang/julia/issues/50074
+[#50105]: https://github.com/JuliaLang/julia/issues/50105
+[#50144]: https://github.com/JuliaLang/julia/issues/50144
+[#50251]: https://github.com/JuliaLang/julia/issues/50251
+[#50471]: https://github.com/JuliaLang/julia/issues/50471
+[#50586]: https://github.com/JuliaLang/julia/issues/50586
+[#50661]: https://github.com/JuliaLang/julia/issues/50661
+[#50795]: https://github.com/JuliaLang/julia/issues/50795
+[#50797]: https://github.com/JuliaLang/julia/issues/50797
+[#50958]: https://github.com/JuliaLang/julia/issues/50958
```
Comment (aviatesk) [inline NEWS.md:229]:
```suggestion
[#50958]: https://github.com/JuliaLang/julia/issues/50958
[#50974]: https://github.com/JuliaLang/julia/issues/50974
```

**Comment by knuesel** [comment]
I think it's missing an entry about #51319 ?

**Comment by JeffBezanson** [comment]
Added an entry to close #51911

**Pull request merged**